_toc.yml

```diff
@@ -1,43 +1,66 @@
 - file: index
 - chapters:
-  - file: community/index
-    sections:
-    - file: community/mission_and_values
-    - file: community/team
-    - file: community/code_of_conduct
-    - file: community/code_of_conduct_reporting
-    - file: community/governance
-    title: Community
+  - file: tutorials/start_index
+    sections: 
+    - file: tutorials/fundamentals/installation
+    - file: tutorials/absolute_beginners
+    - file: tutorials/fundamentals/getting_started
+    - file: plugins/stable/find-and-install-plugin 
+    - file: tutorials/fundamentals/viewer
+    - file: tutorials/fundamentals/layers/index
+    title: Getting started
   - file: tutorials/index
     sections:
-    - file: tutorials/fundamentals/index
-      sections:
-      - file: tutorials/fundamentals/installation
-      - file: tutorials/fundamentals/getting_started
-      - file: tutorials/fundamentals/viewer
-      - file: tutorials/fundamentals/image
-      - file: tutorials/fundamentals/labels
-      - file: tutorials/fundamentals/points
-      - file: tutorials/fundamentals/shapes
-      - file: tutorials/fundamentals/surface
-      - file: tutorials/fundamentals/tracks
-      - file: tutorials/fundamentals/vectors
-    - file: tutorials/applications/index
-      sections:
-      - file: tutorials/applications/annotate_segmentation
-      - file: tutorials/applications/annotate_points
-      - file: tutorials/applications/dask
-      - file: tutorials/applications/napari_imageJ
-      - file: tutorials/applications/cell_tracking
+    #- file: tutorials/fundamentals/index
+    #- file: tutorials/applications/index
+    - file: tutorials/segmentation/index
+    #- file: tutorials/applications/annotate_segmentation
+    - file: tutorials/tracking/index
+    #- file: tutorials/applications/cell_tracking
+    - file: tutorials/applications/annotate_points
+    - file: tutorials/applications/dask
+    - file: tutorials/particle_trackpy
     - file: tutorials/further-resources/napari-workshops
     title: Tutorials
+  - file: howtos/index
+    sections:
+    - file: tutorials/fundamentals/layers/index
+    - file: tutorials/applications/napari_imageJ
+    - file: guides/stable/perfmon
+    - file: guides/stable/docker
+    - file: guides/stable/connecting_events
+    - file: guides/how-to-run-examples
+    - file: guides/remote-based-desktops
+    title: How-tos  
+  - file: guides/stable/index
+    sections:
+    - file: guides/layers
+    #- file: guides/layer_transform_parameters
+    #- file: guides/properties
+    - file: guides/stable/event_loop
+    - file: guides/stable/threading
+    - file: guides/stable/rendering-explanation
+    - file: guides/stable/rendering
+    - file: guides/stable/performance
+    - file: guides/stable/magicgui
+    - file: guides/stable/3D_interactivity
+    #- file: Glossary
+    title: In-depth guides
```

Comment:
Does the ordering of the file/sections/title matter? It would be more intuitive for me for the title to come straight after the file, rather than at the end of the sections listing.

Reply:
I'll have to check the jupyterbook docs, but I agree with you